localization/languages/cs.json

```diff
@@ -187,16 +192,16 @@
         "appMenuFullScreen": "Celá obrazovka", //on some platforms, this string is replaced with one built-in to the OS
         "appMenuFocusMode": "Režim zaměření",
         "appMenuReadingList": "Seznam čtení",
-        "appMenuBookmarks": null, //missing translation
-        "appMenuHistory": null, //missing translation
-        "appMenuDeveloper": "Vývojář",
+        "appMenuBookmarks": "Záložky",
+        "appMenuHistory": "Historie",
+        "appMenuDeveloper": "Nástroje pro vývojáře",
         "appMenuReloadBrowser": "Znovu načíst prohlížeč",
-        "appMenuInspectBrowser": "Prozkoumat prohlížeč",
-        "appMenuInspectPage": "Prozkoumat stránku",
+        "appMenuInspectBrowser": "Prozkoumat DOM a styly prohlížeče",
+        "appMenuInspectPage": "Prozkoumat DOM a styly stránky",
```

Comment:
Could you make sure the meaning of these matches the English version (unless there's a particular reason to alter them)?

Reply:
I think "Inspect DOM and styles of browser" is more understandable for developers than only "Inspect browser". But I can return it to the previous translation, if you don't agree.

Reply:
I think "inspect" is probably sufficient; for comparison, Chrome uses "inspect elements", Firefox uses "inspector", and Safari uses "show web inspector", so I'd prefer to return it to the previous translation.

Reply:
Alright, I returned it to the previous one.